modules/custom/d_p_subscribe_file/src/Forms/SubscribeFileForm.php

```diff
@@ -12,14 +12,15 @@
 
 /**
  * Class SubscribeFileForm
+ *
  * @package Drupal\d_p_subscribe_file\Forms
  */
 class SubscribeFileForm extends FormBase {
 
   /**
    * @var \Drupal\Core\Session\AccountInterface|\Drupal\Core\Session\AnonymousUserSession
    */
-  protected $accountProxy;
+  protected $account_proxy;
```

Comment:
https://www.drupal.org/node/608152 - Methods and class properties should use lowerCamel naming. In Drupal 8, properties of configuration entities are exempt of these conventions. Those properties are allowed to use underscores.

Czyli properties powinniśmy zostawić w camelCase